### PR TITLE
Dashboard plus script change

### PR DESCRIPTION
**Pull request by manuarya27**
Pull request covers - 
1) Changes corresponding to Houssem's new script
2) JSON changes with latest grafana changes

### CODE REVIEW

File: catboost_info/catboost_training.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (jananjoy) [inline catboost_info/catboost_training.json:1]: Format the tag inside the file for improving readability e.g
{
"meta":{
	   "test_sets":[],
		"test_metrics":[],
		"learn_metrics":[{"best_value":"Min","name":"Logloss"}
		 ],
	   "launch_mode":"Train",
	   "parameters":"",
	   "iteration_count":2,
	  "learn_sets":["learn"],
	  "name":"experiment"
 },
"iterations":[
{"learn":[0.1319107819],"iteration":0,"passed_time":0.05867800393,"remaining_time":0.05867800393},
{"learn":[0.04703227997],"iteration":1,"passed_time":0.05900773429,"remaining_time":0}
]}

File: src/app.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+from gen_data_and_simulate_drift import GenerateFakeData, DriftSimulator, DriftIntensity
+from training_script import Classifier
+
+
+if __name__ == '__main__':
+    target = 'Legendary'
+    genertor_fake_data = GenerateFakeData(path_ref_data='pokemon.csv', sample_size = 1000, target=target)
```
Comment (jananjoy) [inline src/app.py:7]: is  sample_size and nb_cols_to_drift constant or configurable parameters?
To remove hard-coded value
SAMPLE_SIZE = 1000
DRIFT_COLUMN = 1
then pass the value to the function/methods

File: src/fake_data_script.py [hunk 3]
```diff
@@ -1,17 +1,45 @@
...
+from pulsar_data_collection.data_capture import (
+    DatabaseLogin,
+    DataCapture,
+    DataWithPrediction,
+)
+
+from gen_data_and_simulate_drift import DriftIntensity, DriftSimulator, GenerateFakeData
+from training_script import Classifier
+
+# if __name__ == "__main__":
```
Comment (jananjoy) [inline src/fake_data_script.py:13]: Remove the commented code if it is not required. For your reference, you can keep it private

File: catboost_info/catboost_training.json (hunk 1 above)
Comment (manuarya27) [reply catboost_info/catboost_training.json:1]: Hi Jananjoy, this json file is automatically generated while building our project. @houdridipas , please confirm.

File: src/fake_data_script.py [hunk 4]
```diff
@@ -1,17 +1,45 @@
...
 
-    model = pkl.load(open("/app/models/kidney_disease.pkl", "rb"))
+    # if the task is classification
+    pok_classifier = Classifier(df_train=sampled_data.train_data,
+                num_features=sampled_data.list_num_col,
+                cat_features=None,
+                target=target,
+                pkl_file_path=f'class_{target}_model.pkl')
+    pok_classifier.train()
+    pok_classifier.serialize()
```
Comment (jananjoy) [inline src/fake_data_script.py:36]: serialize() method internally call pickle API. Do you handle the exception?

File: src/training_script.py [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+    def serialize(self):
+        with open(self.pkl_file_path, 'wb') as file:
+            pickle.dump(self, file)
+
+    def train(self):
+        self.model = CatBoostClassifier(iterations=2,
+                                   learning_rate=1,
+                                   depth=2)
+        self.model.fit(self.df_train[self.num_features + self.cat_features], self.df_train[self.target])
```
[the remark this answers is not in the document]
Comment (jananjoy) [reply src/training_script.py:30]: I think you do not get my comments. Please refer to it 
https://catboost.ai/en/docs/concepts/python-reference_catboostclassifier_fit
To understand the reader of the code. provide code comment for this methods

File: src/app.py (hunk 2 above)
Comment (jananjoy) [reply src/app.py:7]: It is nothing to do with architecture. It is the author's decision. Please refer to PUL76- Pulsar Demo Refactor/Improvement (apply for other files too)
PUL-101 Remove hardcoded values from scripts

File: src/training_script.py (hunk 5 above)
[the remark this answers is not in the document]
Comment (jananjoy) [reply src/training_script.py:30]: This comment corresponds to improving the technical reliability of the code. My comments focus on following info 
https://catboost.ai/en/docs/concepts/python-reference_catboostclassifier_fit

File: src/fake_data_script.py (hunk 4 above)
Comment (manuarya27) [reply src/fake_data_script.py:36]: Have added the exception handling in trainingscript.py.

**Review by houdridipas — APPROVED** [review]

**Pull request merged**